src/pybind/mgr/orchestrator.py

```diff
@@ -5,10 +5,16 @@
 Please see the ceph-mgr module developer's guide for more information.
 """
 
+from typing import TypeVar, Generic, List, Dict
```

Comment:
`typing` was introduced by python 3.5. probably, we can catch `ImportError` and fallback to `object` if `typing` is not around? we can always use mypy in our own development environment if we want to perform some static analysis with python2. 

see http://pulpito.ceph.com/sage-2019-01-01_22:42:55-rados-wip-sage2-testing-2019-01-01-1414-distro-basic-smithi/

Reply:
```
2019-01-01T23:55:48.808 INFO:tasks.ceph.mgr.x.smithi093.stderr:  File "/usr/lib64/ceph/mgr/ansible/__init__.py", line 5, in <module>
2019-01-01T23:55:48.808 INFO:tasks.ceph.mgr.x.smithi093.stderr:    from .module import Module
2019-01-01T23:55:48.808 INFO:tasks.ceph.mgr.x.smithi093.stderr:  File "/usr/lib64/ceph/mgr/ansible/module.py", line 10, in <module>
2019-01-01T23:55:48.808 INFO:tasks.ceph.mgr.x.smithi093.stderr:    import orchestrator
2019-01-01T23:55:48.809 INFO:tasks.ceph.mgr.x.smithi093.stderr:  File "/usr/lib64/ceph/mgr/orchestrator.py", line 8, in <module>
2019-01-01T23:55:48.809 INFO:tasks.ceph.mgr.x.smithi093.stderr:    from typing import TypeVar, Generic, List, Dict
2019-01-01T23:55:48.809 INFO:tasks.ceph.mgr.x.smithi093.stderr:ImportError: No module named typing
```

One problem is that catching `ImportError` breaks the IDE support. Guess I need to simplify this.

Reply:
> One problem is that catching ImportError breaks the IDE support. Guess I need to simplify this.

@sebastian-philipp is it still true?

Reply:
nope. I fixed it

Reply:
cool, so can we remove the "DNM" label?

Reply:
sure. sorry.